roles/confluent.variables/vars/main.yml

```diff
@@ -12,10 +12,13 @@ zookeeper_default_log_dir: /var/log/kafka
 zookeeper:
   server_start_file: "{{ binary_base_path }}/bin/zookeeper-server-start"
   config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/zookeeper.properties"
-  systemd_file: "{{systemd_base_dir}}/{{zookeeper_service_name}}.service"
-  systemd_override: /etc/systemd/system/{{zookeeper_service_name}}.service.d/override.conf
+  # systemd_file: "{{systemd_base_dir}}/{{zookeeper_service_name}}.service"
+  # systemd_override: /etc/systemd/system/{{zookeeper_service_name}}.service.d/override.conf
   log4j_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/zookeeper-log4j.properties"
 
+  systemd_file: "/usr/local/confluent/lib/systemd/system/{{zookeeper_service_name}}.service"
+  systemd_override: "/usr/local/confluent/lib/systemd/system/{{zookeeper_service_name}}.service.d/override.conf"
```

Comment:
so we most likely only want the directory customizable here, not the full file path (i think).
We can move the `systemd_base_dir` to defaults/main.yml
And create a `systemd_override_dir` in defaults/main.yml

And those will need to be in your inventory vars.

Reply:
done